src/app/dashboard/page.tsx

```
@@ -1,51 +1,272 @@
+  const competitionCampaigns = useMemo(
+    () =>
+      allCampaigns.filter(
+        (c) =>
+          !selectedCompetition?.competition_id ||
+          c.competition_id === selectedCompetition.competition_id,
+      ),
+    [allCampaigns, selectedCompetition?.competition_id],
+  );
+
+  const sortedCampaigns = useMemo(() => {
+    let copy = [...competitionCampaigns].filter((c) => c.status === "published");
```

ESLint

[error] 'copy' is never reassigned. Use 'const' instead.

(prefer-const)